index.bs

```diff
@@ -77,6 +81,22 @@ This key must be present. This property can be localized.
 
 This key must be present.
 
+### Key `permissions`
+
+This key may be present.
+
+### Key `optional_permissions`
+
+This key may be present.
+
+### Key `host_permissions`
+
+This key may be present.
+
+### Key `optional_host_permissions`
+
+This key may be present.
+
```

Comment:
Nit: We may want to consider sorting the list of keys alphabetically or by another well-defined scheme.

Reply:
Changed the casing of the Manifest V3 mention.

Reply:
I prefaced this with "nit" because it was a small, relatively unimportant observation. I was mostly just thinking out loud. I mostly agree with the intent behind grouping related keys.

That said,  but for readers that aren't clear on the association between proximity-based groups I worry that it can end up feeling like a random collection rather than an intentionally curated list. One way we could address this is by making groups part of the heading structure of the document, but I don't feel strongly enough about it to tackle that concern in this PR or likely even this year.